deprecation.rst

```diff
@@ -17,3 +17,25 @@ no version bumps are expected.
 
 The current list of deprecated EAPIs is stored as ``eapis-deprecated``
 in ``metadata/layout.conf``.
+
+
+.. index:: EAPI; banned
+
+Banned EAPIs
+------------
+:PG: 1002
+:Source: Council
+:Reference: `Council-approved EAPIs`_
+:Reported: by pkgcheck and repoman
+
+Banned EAPIs must not be used by any new ebuilds.  Existing packages
+should proactively be migrated to new EAPIs.  The only allowed exception
+is when the new ebuild is a result of revision bump due to dependency
+change or an important bug fix.
```

Comment:
I don't think that there is any such exception. At least for EAPI 4, new ebuilds are unconditionally banned:
https://projects.gentoo.org/council/meeting-logs/20180408-summary.txt
Which makes sense, given the fact that the ban is enacted only after a deprecation period of several years.

For EAPIs 0 and 3 we had an exception for "the security team and only the security team" but I think this is no longer relevant:
https://projects.gentoo.org/council/meeting-logs/20160110-summary.txt

Reply:
Still, I would permit revbumps due to dependency changes. It's silly to require other people to update some maintainer's packages just because they have to revbump them. That said, it depends if we call `git mv` a new ebuild ;-).

Reply:
Typically such packages don't have active maintainers, so other people doing the EAPI bump isn't so silly.

If we don't apply stricter conditions for banned than for deprecated, then the whole concept of banned EAPIs doesn't make sense.

There would still be time to bring this up for the concil meeting next sunday, i.e., ask for the ban for EAPI 4 to be loosened (agenda is still pending, and chairman is in UTC-8 time zone).

Reply:
Ok, I will bring that up to the ml.